bugfix/redshift-large-int (#117)
fivetran/dbt_fivetran_log: CHANGELOG.md

# dbt_fivetran_log v1.6.0

> [PR #117](https://github.com/fivetran/dbt_fivetran_log/pull/117) includes the following updates as a result of users encountering numeric counts exceeding the limit of a standard integer. Therefore, the fields were required to be updated to a bigint in order to avoid integer out of range errors:

Reviewer: ```suggestion
[PR #117](https://github.com/fivetran/dbt_fivetran_log/pull/117) includes the following updates as a result of users encountering numeric counts exceeding the limit of a standard integer. Therefore, these fields were required to be cast as `bigint` in order to avoid "integer out of range" errors:
```

## Breaking Changes

> > ⚠️ Since the following changes result in a field changing datatype, we recommend running a `--full-refresh` after upgrading to this version to avoid possibly incremental failures.

Reviewer 2: small typo
```suggestion
> ⚠️ Since the following changes result in a field changing datatype, we recommend running a `--full-refresh` after upgrading to this version to avoid possible incremental failures.
```

- The following fields in the `fivetran_platform__audit_table` model have been updated to be cast as `dbt.type_bigint()` (previously was `dbt.type_int()`)
  - `sum_rows_replaced_or_inserted`
  - `sum_rows_updated`
  - `sum_rows_deleted`

## Bug Fixes
- The following fields in the `fivetran_platform__connector_daily_events` model have been updated to be cast as `dbt.type_bigint()` (previously was `dbt.type_int()`)
  - `count_api_calls`
  - `count_record_modifications`
  - `count_schema_changes`

## Under the Hood

> - Modified `log` seed data within the integration tests folder to ensure a large integers are being tested as part of our integration tests.

Reviewer: ```suggestion
- Modified `log` seed data within the integration tests folder to ensure that large integers are being tested as part of our integration tests.
```
